stimela/kitchen/step.py

```diff
@@ -8,7 +8,7 @@
 
 from stimela.config import EmptyDictDefault, EmptyListDefault
 import stimela
-from stimela import log_exception, stimelogging
+from stimela import log_exception, stimelogging, task_stats
 from stimela.backends import StimelaBackendSchema, runner
 from stimela.exceptions import *
```

Comment:
Not from this PR, but we need to avoid this (`import *`). It makes debugging unnecessarily more difficult